src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -17,9 +17,19 @@ import * as sqlExtHostType from 'sql/workbench/api/common/sqlExtHostTypes';
 import { TextWithIconColumn } from 'sql/base/browser/ui/table/plugins/textWithIconColumn';
 
 export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanPropertiesViewBase {
+	private static readonly _topTitleColumnHeader = localize('nodePropertyViewNameValueColumnTopHeader', "Value (Top Plan)");
```

Comment:
I'd just make these constants at the module level, especially for strings only used within this module.